src/workers/parser/utils/helpers.js

```diff
@@ -78,7 +78,9 @@ export function getVariables(dec: Node) {
       return [];
     }
 
-    return dec.id.elements.map(element => {
+    // NOTE: it's possible that an element is empty
+    // e.g. const [, a] = arr
+    return dec.id.elements.filter(element => element).map(element => {
```

Comment:
I use this at times and still forget that it's possible! 👍 Love the addition of the test fixture too.